docs/CHANGELOG.rst

```diff
@@ -37,6 +37,7 @@ Fixed
 * Fix reads filtering processes for cases where input read file names contain whitespace
 * Add additional filtering option to STAR aligner
 * Fix bbduk-star-htseq_count workflow
+* Fix cuffnorm process
```

Comment:
Be more descriptive here, i.e. `Use sample names as labels in cuffnorm and save ___ files to its output.`

http://keepachangelog.com/en/0.3.0/